src/Stratis.Bitcoin.Features.PoA/PoAFeature.cs

```diff
@@ -86,21 +89,26 @@ public override Task InitializeAsync()
 
             this.federationManager.Initialize();
 
+            if (((PoAConsensusOptions)this.network.Consensus.Options).VotingEnabled)
+            {
+                this.votingManager.Initialize();
```

Comment:
I thin this is problematic to put in the feature as its part of consensus, this should likely go in the PoARulesEngine

Reply:
It has nothing to do with validation of a block, why would it go to the rules engine?

Reply:
hmm... fair point, so how do you verify a new member is no allowed to create blocks?

Reply:
`no allowed to create blocks`

did you mean 'now'?